lib/shipit.rb

```diff
@@ -120,7 +120,7 @@ def github(organization: github_default_organization)
 
   def github_default_organization
     return nil unless secrets&.github
-    org = secrets.github.keys.first
+    org = secrets.github.keys.first.to_s
```

Comment:
There are other places where the secrets are accessed, e.g. https://github.com/Shopify/shipit-engine/blob/1e90347cfd47593db21e84626d4864e5cd7c2e3e/lib/shipit.rb#L107-L119 and https://github.com/Shopify/shipit-engine/blob/1e90347cfd47593db21e84626d4864e5cd7c2e3e/lib/shipit.rb#L129. Did you verify that these don't need to be string/symbol based keys?